pkgs/development/octave-modules/zeromq/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildOctavePackage
+, stdenv
+, fetchurl
+, zeromq
+}:
+
+buildOctavePackage rec {
+  pname = "zeromq";
+  version = "1.5.2";
+
+  src = fetchurl {
+    url = "https://octave.sourceforge.io/download.php?package=${pname}-${version}.tar.gz";
+    sha256 = "18h1039ri7dr37jv20cvj5vhw7b57frrda0hhbvlgixinbqmn9j7";
+  };
+
+  buildInputs = [
+    zeromq
+  ];
+
+  meta = with stdenv.lib; {
```

Comment:
A recent change to contributing now says that this should be lib.

Reply:
@SuperSandro2000, can you link to the change? I don't see any way to change `with stdenv.lib` to `with lib` without having to pass `lib` in somewhere else. This would also mean I don't need to pass `stdenv` to any of these package definitions. Is that what you are suggesting?

Reply:
https://github.com/NixOS/nixpkgs/pull/109455

>  This would also mean I don't need to pass stdenv to any of these package definitions. Is that what you are suggesting?

Yes